src/components/comments.js

```diff
@@ -57,7 +58,30 @@ const userRef = {
     })
     
     }  
-
+    
+    const handleDelete = (event) => {
+        event.preventDefault()
+      setCommentId(event.target.className)
+      setIsLoading(true);
+        setErr(null)
+        fetch(`https://hannybees-news-app.herokuapp.com/api/comments/${commentId}`, {
+                method: 'DELETE',
+    }).then(() =>{
+        fetch(`https://hannybees-news-app.herokuapp.com/api/articles${articleIdRoute}/comments`).then((response) => {
+        return response.json();
+    }).then((data) => {
+        setCommentData(() => {
+            return data
+             
+         })
+         setIsLoading(false);
+     })
+ },[]);
```

Comment:
once you make the other changes, you may need to add commentData as a dependency in your useEffect array if it doesn't work straight away!